core/src/main/scala/org/apache/spark/sql/SnappyParser.scala

```diff
@@ -768,7 +768,7 @@ class SnappyParser(session: SnappySession)
   }
 
   protected final def relationWithExternal: Rule1[LogicalPlan] = rule {
-    ((relationFactor | inlineTable) ~ tableValuedFunctionExpressions.?) ~>
+    ((inlineTable | relationFactor) ~ tableValuedFunctionExpressions.?) ~>
```

Comment:
Will do. Might be a side effect of some debugging.

Reply:
@rishitesh this was fixing a bug. Do not revert. There was a test which was failing. I do not remeber though. @hemanthmeka might recall this.

Reply:
Yes, if done to fix some bug then that should be clearly noted here. Why ordering matters. And is there any other cleaner solution rather than depending on ordering.

Reply:
I am reverting it now. Anyway, we have around 140 errors to fix in Spark suite. Once we find the test case which required the change can describe it and change again.